src/site/markdown/getting-started.md

```diff
@@ -17,58 +17,66 @@
 
 ## Getting Started
 
-To on-board incremental Maven you need to complete several steps:
+To onboard incremental Maven you need to complete several steps:
 
 * Declare caching extension in your project (either in `pom.xml` or `.mvn/extensions.xml`)
-* Add `maven-build-cache-config.xml` cache config in `.mvn/` (optional) to customize default behavior
-* Validate build results and iteratively, adjust config to properly reflect project specifics
+* Add `maven-build-cache-config.xml` cache config in `.mvn/` (optional) to customize the default behavior
+* Validate build results and iteratively adjust config to reflect project specifics properly
 * Setup remote cache (optional)
 
-### Declaring build cache extension
+### Declaring build-cache extension
 
 ```xml
+
 <extension>
     <groupId>org.apache.maven.extensions</groupId>
     <artifactId>maven-build-cache-extension</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0</version>
 </extension>
 ```
 
-either in `pom.xml`'s `<project>/<build>/<extensions>` or in `.mvn/extensions.xml`'s `<extensions>`
+either in `pom.xml`'s `<project>/<build>/<extensions>` or in `.mvn/extensions.xml`'s `<extensions>`. Using core
+extension model (`.mvn/extensions.xml` file) is preferable as it allows better access to maven APIs and could allow
+more sophisticated optimizations in the future.
 
-### Adding build cache config
+### Adding build-cache config
 
 Copy template config [`maven-build-cache-config.xml`](../resources/maven-build-cache-config.xml)
 to [`.mvn/`](https://maven.apache.org/configure.html) directory of your project.  
-To get overall understanding of build cache machinery, it is recommended to review the config and read comments. In typical
-scenario you need to:
+To understand the caching machinery, review the config and read the comments. In a typical scenario, need to:
 
-* Exclude unstable, temporary files or environment specific files
-* Add plugins reconciliation rules – add critical plugins parameters to reconciliation
-* Configure precise source code files selectors. Though source code locations discovered automatically from project and plugins config,
-  there might be edge cases.
-* Add remote cache location (if remote cache is used)
+* Exclude unstable, temporary files or environment-specific files
+* Add critical plugins parameters to runtime reconciliation
+* Configure precise source code file selectors. Though source code locations are discovered automatically from project
+  and
+  plugin configs, there might be edge cases.
+* Configure remote cache (if using the remote cache)
 
-### Adjusting build cache config
+### Adjusting build-cache config
 
-Having extension run usual command, like `mvn package`. Verify the caching engine is activated:
+Having the extension configured, run a usual command like `mvn package`. Verify the caching engine is activated:
```

Comment:
```
Once the extension is configured, run a usual command like `mvn package` and verify the caching engine is activated:
```

Reply:
Changed to `After configuring the extension, run a usual command, for example, `mvn package`, and verify the caching engine is activated:` to avoid using the passive voice